Documentation/async-disaster-recovery.md

```diff
@@ -0,0 +1,454 @@
+
+  ```bash
+    kubectl create -f examples/rbd/storageclass.yaml --context=cluster-2
+  ```
+
+  > ```bash
+  > storageclass.storage.k8s.io/csi-rbd-sc created
+  > ```
+
+  * Create VolumeReplicationClass on the secondary cluster
+
+  ```bash
+        cat <<EOF | kubectl --context=cluster-2 apply -f -
+        apiVersion: replication.storage.openshift.io/v1alpha1
+        kind: VolumeReplicationClass
+        metadata:
+          name: rbd-volumereplicationclass
+        spec:
+          provisioner: rook-ceph.rbd.csi.ceph.com
+          parameters:
+            mirroringMode: snapshot
+            replication.storage.openshift.io/replication-secret-name: rook-csi-rbd-provisioner
+            replication.storage.openshift.io/replication-secret-namespace: rook-ceph
+        EOF
+  ```
+
+  > ```bash
+  > volumereplicationclass.replication.storage.openshift.io/rbd-volumereplicationclass created
+  > ```
+
+  * If Persistent Volumes and Claims are created manually on the secondary cluster,
+     remove the `claimRef` on the backed up PV objects in yaml files; so that the
+     PV can get bound to the new claim on the secondary cluster.
+
+  ```yaml
+    ...
+    spec:
+        accessModes:
+        - ReadWriteOnce
+        capacity:
+          storage: 1Gi
+        claimRef:
+          apiVersion: v1
+          kind: PersistentVolumeClaim
+          name: rbd-pvc
+          namespace: default
+          resourceVersion: "64252"
+          uid: 65dc0aac-5e15-4474-90f4-7a3532c621ec
+        csi:
+    ...
+  ```
+
+* Apply the Persistent Volume backup from the primary cluster
+
+```bash
+    kubectl create -f pv-backup.yaml --context=cluster-2
+```
+
+> ```bash
+> persistentvolume/pvc-65dc0aac-5e15-4474-90f4-7a3532c621ec created
+> ```
+
+* Apply the Persistent Volume claim from the restored backup
+
+```bash
+    kubectl create -f pvc-backup.yaml --context=cluster-2
+```
+
+> ```bash
+> persistentvolumeclaim/rbd-pvc created
+> ```
+
+```bash
+  kubectl get pvc --context=cluster-2
+```
+
+> ```bash
+> NAME      STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS      AGE
+> rbd-pvc   Bound    pvc-65dc0aac-5e15-4474-90f4-7a3532c621ec   1Gi        RWO            csi-rbd-sc   44s
+> ```
+
+## Planned Migration
+
+> Use cases: Datacenter maintenance, Technology refresh, Disaster avoidance, etc.
+
+### Failover
+
+The failover operation is the process of switching production to a
+ backup facility (normally your recovery site). In the case of Failover,
+ access to the image on the primary site should be stopped.
+The image should now be made *primary* on the secondary cluster so that
+ the access can be resumed there.
+
+:memo: As mentioned in the pre-requisites, periodic or one time backup of
+ the application should be available for restore on the secondary site (cluster-b).
+
+Follow the below steps for planned migration of workload from primary
+ cluster to secondary cluster:
+
```

Comment:
IMO, we should provide `kubectl` commands for the steps mentioned below, where ever possible.

Reply:
Thanks for the suggestion! Although adding `kubectl` commands for every step will be more descriptive, but it will come with the downside of unnecessarily repeating the redundant steps. 
For e.g: Updating the VolumeReplication CR is a highly repetitive operation throughout the failover and failback steps, which we have already mentioned above how to do [here](https://github.com/rook/rook/pull/8411/files#diff-1b0b287cce986a56c15d68e1b16ad5941350b0fee93c11ff1de7c6d829456b05R192). 
Similarly, descriptions of steps to do all the other operations in failover and failback are already mentioned on the doc before coming to the `failover and failback steps` section. 
Thus, I think it will be better if we keep it descriptive and concise at the same time. 
Thoughts? @Madhu-1 @sp98 @leseb @travisn

Reply:
If we're not going to provide the commands, we should refer back to some earlier section where commands were provided. So hopefully we can avoid duplicating common steps, but still show the commands.

Reply:
Thanks for the suggestion! Now each step refers to the section where commands are provided.